x-pack/plugins/inference/common/http_apis.ts

```diff
@@ -15,3 +15,7 @@ export type ChatCompleteRequestBody = {
   messages: Message[];
   functionCalling?: FunctionCallingMode;
 } & ToolOptions;
+
+export interface GetConnectorsResponseBody {
```

Comment:
Why are these types not extracted to the `inference-common` package? Is it because they aren't used outside of the `inference` plugin?

Reply:
Correct, those are the types for the response payloads of the http APIs, it's only used internally by the plugin.